recipe/build.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+export LDFLAGS="${LDFLAGS} -llapack -lblas -lfftw3 -lfftw3f"
+export LDFLAGS_LD="${LDFLAGS_LD} -llapack -lblas -lfftw3 -lfftw3f" 
```

Comment:
As I manually have to specify the libraries, there seems to be something wrong with how they are linked in the build.